### PR TITLE
add openbasedir option "parent of documentroot" - master/v0.10.0

### PR DESCRIPTION
**Pull request by florianschroen**
refers to
 https://github.com/Froxlor/Froxlor/issues/515

+ little refactoring in cron_tasks.inc.http.10.apache.php for better
  code maintenance

### CODE REVIEW

File: scripts/jobs/cron_tasks.inc.http.10.apache.php [hunk 1]
```diff
@@ -557,18 +557,35 @@ protected function composePhpOptions($domain, $ssl_vhost = false)
...
 			if ($domain['openbasedir'] == '1') {
-				if ($domain['openbasedir_path'] == '1' || strstr($domain['documentroot'], ":") !== false) {
-					$_phpappendopenbasedir = appendOpenBasedirPath($domain['customerroot'], true);
+				/***
+					0   - documentroot
+					1   - customerroot
+					2   - parent of documentroot
+					*:* - customerroot
+				***/
+				if ($this->_domain['openbasedir_path'] == '2'
```
Comment (d00p) [inline scripts/jobs/cron_tasks.inc.http.10.apache.php]: there is no class-member called "_domains"...did you just copy/paste from fcgid/fpm?

**Comment by d00p** [comment]
no more responses, closed for now

**Comment by makuser** [comment]
Oh maaaaan. C'mon...

**Comment by d00p** [comment]
> Oh maaaaan. C'mon...

Again, time time time, if people start something and then don't respond anymore I can't help it, cannot do everything

**Comment by florianschroen** [comment]
Sorry, hadn't time. That's right. Will do this when I have time to get into new code of api and start a new pr.
(haven't seen d00p's msg with api functions).

**Pull request closed without merging**